scripts/psp-makepkg

```
@@ -5,20 +5,22 @@ if [ -z "${PSPDEV}" ]; then
     export PSPDEV=/usr/local/pspdev
```

I would like to error out here instead of setting some arbitrary default.

---

Good point. This made sense when my assumption was that users would run psp-pacman with sudo directly. We've since made this never needed.

---

Even if the user runs it with sudo, they might have it in an entirely different place (`/opt` or `/home/user` are much more common to me).